packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -1603,6 +1603,48 @@ void main() {
       expect(viewsController.gesturesAccepted[currentViewId + 1], 0);
     });
 
+    testWidgets('UiKitView rejects gestures absorbed by siblings if the touch is outside of the platform view bounds but inside platform view frame', (WidgetTester tester) async {
+      // UIKitView has bounds of (0, 0, 300, 500)
+      // UIKitView has frame of (0, 100, 300, 500)
```

Comment:
nit: I was a bit confused by the bounds/frame terminology in this comment, maybe say:

// UiKitView is positioned at (left=0, top=100, right=300, bottom=600)
// Opaque container is on top of the UiKitView positioned at (left=0, top=500, right=300, bottom=600)
// Touch on (550, 150) is expected to be absorbed by the container.

Reply:
done